libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -620,42 +625,54 @@ private Optional<JsonNode> generateNinoClaim(
 
             return Optional.empty();
         }
+        var mappedNino = nino.stream().map(this::mapNinoToSocialSecurityRecord).toList();
 
-        return Optional.of(ninoNode);
+        return Optional.of(mappedNino);
     }
 
-    private Optional<JsonNode> generatePassportClaim(List<VerifiableCredential> vcs)
-            throws HttpResponseExceptionWithErrorBody {
+    private SocialSecurityRecord mapNinoToSocialSecurityRecord(SocialSecurityRecordDetails nino) {
```

Comment:
(Ditto again - it would be nice to move to using the vocab type and remove this mapping :) )